components/script/dom/htmlscriptelement.rs

```diff
@@ -308,6 +308,7 @@ pub(crate) struct ScriptOrigin {
     fetch_options: ScriptFetchOptions,
     type_: ScriptType,
     unminified_dir: Option<String>,
+    import_map_result: Option<Fallible<ImportMap>>,
```

Comment:
In the [execute()](https://github.com/servo/servo/blob/ef5e9b5f4dfcf9face95ba01b648f194084a4262/components/script/dom/htmlscriptelement.rs#L1046), some information that should be from the `HTMLScriptElement` is stored in the `ScriptOrigin` instead of the script element itself, that's why I put the `import_map_result` in `ScriptOrigin` as well. But I think we should eventually follow the spec and add a `result` member in the `HTMLScriptElement`?

- https://html.spec.whatwg.org/multipage/scripting.html#execute-the-script-element
- https://html.spec.whatwg.org/multipage/scripting.html#concept-script-result

Reply:
This can be one `Fallible` with default to `Error::NotFound` probably.
```suggestion
    import_map_result: Fallible<ImportMap>,
```